hop/io.py

```diff
@@ -27,7 +28,7 @@ class Stream(object):
     stream connection is opened, it will use defaults specified here.
 
     Args:
-        auth: A `bool` or `Auth` instance. Defaults to loading from `auth.load_auth()`
```

Comment:
Most of this should be reverted back, minus the function name `get_config_path()`

Reply:
I changed this comment to "auth" and "Auth" instead of "config" and "Config"